crdb/tx.go

```diff
@@ -50,6 +50,17 @@ func ExecuteTx(db *sql.DB, fn func(*sql.Tx) error) (err error) {
 	if err != nil {
 		return err
 	}
+	return ExecuteInTx(tx, func() error { return fn(tx) })
+}
+
+type Tx interface {
+	Exec(query string, args ...interface{}) (sql.Result, error)
+	Commit() error
+	Rollback() error
+}
+
+// ExecuteInTx runs fn inside tx which should already have begun.
```

Comment:
What happens if statements have already been executed in the transaction before `ExecuteInTx` is called? I don't think those statements will be re-executed. Not sure who on the Cockroach side would know. @bdarnell?